projects/common/src/telemetry/user-telemetry-impl.service.ts

```diff
@@ -71,6 +71,21 @@ export class UserTelemetryImplService extends UserTelemetryService {
       );
   }
 
+  private getDifferenceInDays(timeParamValue: string | undefined): number | undefined {
```

Comment:
Not a blocker, but try adding a test for this function before merging to increase coverage a bit.

Reply:
Done.